### PR TITLE
Recovery link

### PR DESCRIPTION
**Pull request by adriandumavitagroup**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,5 @@ docker/translations/node_modules
 
 # translations from backend
 /src/assets/i18n/dist
+
+.angular
```
Comment (danielrus) [inline .gitignore:61]: duplicate.. it's already there

File: src/app/shared/modals/search-case-modal/search-case-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-search-case-modal',
+  templateUrl: './search-case-modal.component.html',
+  styleUrls: ['./search-case-modal.component.scss'],
+})
+export class SearchCaseModalComponent {
+  filtersForm = new FormGroup({});
```
Comment (danielrus) [inline src/app/shared/modals/search-case-modal/search-case-modal.component.ts:13]: you're using nameUuidEpidNumberLike in the template, but it's not defined here in the form

File: src/theme/form-field.scss [hunk 3]
```diff
@@ -98,7 +98,7 @@
       .mat-form-field-wrapper {
         padding-top: 0;
         padding-bottom: 0;
-        margin: 0.5rem 0;
+        margin: 0;
```
Comment (danielrus) [inline src/theme/form-field.scss]: this breaks the design for all the forms

File: src/app/shared/card/card-vaccination/card-vaccination.component.html [hunk 4]
```diff
@@ -1,10 +1,23 @@
...
+  <div class="header">
+    <p class="vaccine-name">ULF7SS</p>
+  </div>
   <p class="vaccine-manufacturer">Oxford–AstraZeneca COVID-19 vaccine</p>
   <div class="middle">
-    <p>Viraler Vector</p>
-    <div class="dose-number">
-      2
+    <div class="left">
+      <p class="vaccine-type">Viraler Vector</p>
```
Comment (sandorbanvitagroup) [inline src/app/shared/card/card-vaccination/card-vaccination.component.html:9]: Is this constant? Does it require a translation?

**Review by sandorbanvitagroup — DISMISSED** [review]

**Review by danielrus — DISMISSED** [review]

**Review by sandorbanvitagroup — APPROVED** [review]

**Review by danielrus — APPROVED** [review]

**Pull request merged**